types/redux-persist-webextension-storage/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for redux-persist-webextension-storage 1.0
+// Project: https://github.com/ssorallen/redux-persist-webextension-storage
+// Definitions by: Marton Lederer <https://github.com/martonlederer>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface StorageInterface {
+  getItem(key: string): Promise<any>;
+  removeItem(key: string): Promise<void>;
+  setItem(key: string, value: any): Promise<void>;
+}
+
+export function localStorage(): StorageInterface;
+export function syncStorage(): StorageInterface;
+export {};
```

Comment:
```suggestion
```
You don't need this one, just use `export` keyword for `localStorage`, `syncStorage` and `StorageInterface` (unless the intent is to not export `StorageInterface`)

Reply:
No I don't want to export that, it's just a helper

Reply:
cool, so just disregard this comment, thx!